WorkflowUI/Sources/ViewControllerDescription/ViewControllerDescription.swift

```diff
@@ -36,6 +36,7 @@
     public struct ViewControllerDescription {
         private let viewControllerType: UIViewController.Type
```

Comment:
We don't strictly need this type anymore, it's now only used for printing the precondition error message. But to avoid breaking any clients that pass it in explicitly, I've left the initializer param in there. Open to removing it entirely.

Reply:
I think it's useful for the error message (IMO).